roofit/roofitcore/inc/RooAbsReal.h

```diff
@@ -463,7 +463,35 @@ class RooAbsReal : public RooAbsArg {
 
   //--------------------------------------------------------------------
 
- protected:
+  // Experimental AD feature functions
+  protected:
+  /// the variable that stores the result expression for the object.
+  std::string _adResult = ""; ///<!
+
+  public:
+  virtual std::string translate(std::string & /* globalScope */, std::vector<std::string> & /* preFuncDecls */);
```

Comment:
```suggestion
  virtual std::string translate(std::string & /* globalScope */, std::vector<std::string> const& /* preFuncDecls */);
```
Should `preFuncDecls` not be taken as a `const` reference here?